### ISSUE TITLE
Enhancement: Generalized Tool Hints

### ISSUE THREAD

**Issue by TehTurk** [Frooxius/NeosPublic#982]
Figured this will come along but due to some discussions in the discord today I feel like having a generalized tool hint system would be beneficial as Users can use this and this can also be integrated into the UI. This also alleviates the strain on vetern players having to educste users consistently. The current system has some issues and doesn't fit all cases so I propose improving this system.

**Comment by Frooxius** [comment]
Yes, this is in the pipeline, not 100 % sure when it's going to be added yet.

**Comment by TehTurk** [comment]
Yup :) just noting it down for future things or considerations.

**Comment by Cynosurae** [comment]
I was in the middle of writing a enhancement request for this when I came across this one, so I'll just echo the sentiment here instead.

UI hints ("tooltips" in the general UI definition of the term, not the Neos usage of the term) should pop up when hovering over the buttons in the upper right of the Inventory screen and Scene Inspector. This would be super useful for onboarding new players, particularly new players that might not have someone to show them around, and even small enhancements like this do wonders for user-friendliness.

Users who do not log in frequently will likely not remember what each button does- even if documented in a tutorial, these infrequent players would probably prefer a quick reminder instead of a full tutorial.

A similar UI hint would be helpful in the File Browser, as long filenames are completely unreadable in their "tiles" at a glance unless you click on each individual file to see its full name pop up at the top of the browser.

**Comment by UnderSampled** [comment]
This is something I've noticed, missed, and desired every time I have logged in. I am one of those infrequent players, and if asked, this is the one thing I'd say Neos needs. Please prioritize this higher!
